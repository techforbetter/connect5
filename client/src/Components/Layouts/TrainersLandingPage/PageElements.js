import styled from "styled-components";
import { Link } from "react-router-dom";

const Container = styled.div`
  display: flex;
  flex-direction: column;
  width: 80%;
  margin: 0 auto;
  text-align: center;
  height: 100%;
  margin-top: 120px;
  max-width: 600px
`;

const Heading = styled.h1`
  font-size: 1.7em;
  margin: 0 auto;
  color: var(--line-color);
`;
const Buttons = styled.div`
  display: flex;
  flex-direction: column;
  margin: 0 auto;
  width: 100%;
`;


const Paragraph = styled.p`
  color: #555
  font-weight: 400;
  margin-top: 50px;
  font-size: 0.85em
  text-align: center;
  margin-bottom: 40px
`;

const Button = styled(Link)`
  background: none;
  line-height: 45px;
  margin: 10px 0;
  border-radius: 5px;
  text-transform: uppercase;
  font-weight: 700;
  font-size: 1.4em
  color: black;
  text-decoration: none;
  background-color: var(--large-button-background);
  color: var(--large-button-text);
`;

export {
  Button,
  Paragraph,
  Heading,
  Container,
  Buttons,
};
