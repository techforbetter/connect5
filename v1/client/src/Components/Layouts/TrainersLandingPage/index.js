import React from "react";

import {
  Button, Paragraph, Heading, Container, Buttons,
} from "./PageElements";

const TrainersLandingPage = () => (
  <Container>
    <Heading>Welcome</Heading>
    <Heading>Connect 5 Trainer</Heading>
    <Paragraph>
      If you are a Connect 5 trainer please register a new account or, if you have already
      registered, log in using your existing details.
    </Paragraph>
    <Buttons>
      <Button to="/trainer/register">Sign up</Button>
      <Button to="/trainer/login">Login</Button>
    </Buttons>
  </Container>
);

export default TrainersLandingPage;
