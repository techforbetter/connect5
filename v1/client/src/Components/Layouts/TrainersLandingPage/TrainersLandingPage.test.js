import React from "react";
import renderer from "react-test-renderer";
import { StaticRouter } from "react-router-dom";

import TrainersLandingPage from "./index";

test("Header matches snapshot", () => {
  const context = {};

  const component = renderer.create(
    <StaticRouter context={context}>
      <TrainersLandingPage />
    </StaticRouter>,
  );
  expect(component).toMatchSnapshot();
});
