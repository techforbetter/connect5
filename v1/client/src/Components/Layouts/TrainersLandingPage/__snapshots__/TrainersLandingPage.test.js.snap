// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches snapshot 1`] = `
<div
  className="sc-bdVaJa eilYpv"
>
  <h1
    className="sc-bwzfXH bVGCDQ"
  >
    Welcome
  </h1>
  <h1
    className="sc-bwzfXH bVGCDQ"
  >
    Connect 5 Trainer
  </h1>
  <p
    className="sc-bxivhb cYGKpg"
  >
    If you are a Connect 5 trainer please register a new account or, if you have already registered, log in using your existing details.
  </p>
  <div
    className="sc-htpNat kXvQrY"
  >
    <a
      className="sc-ifAKCX ZsYgY"
      href="/trainer/register"
      onClick={[Function]}
    >
      Sign up
    </a>
    <a
      className="sc-ifAKCX ZsYgY"
      href="/trainer/login"
      onClick={[Function]}
    >
      Login
    </a>
  </div>
</div>
`;
